nsnet2/nsnet2.js

```diff
@@ -0,0 +1,182 @@
+
+'use strict';
+
+function sizeOfShape(shape) {
+  return shape.reduce((a, b) => {
+    return a * b;
+  });
+}
+
+export class NSNet2 {
```

Comment:
@wchao1115, could you please help review this NSNet2 implementation? If the code looks good, I could submit a patch to update the NSNet2 example code in the [explainer](https://github.com/webmachinelearning/webnn/blob/master/explainer.md#key-scenarios) according to this one. 

What do you think?

Reply:
looks great! Thanks!